lib/clean_test.py

```diff
@@ -36,3 +37,58 @@ def test_remove_future_dates(self):
                 columns=columns,
             ),
         )
+
+
+class FoatIntStr(unittest.TestCase):
```

Comment:
All classes inherit from `unittest.TesetCase` should have "TestCase" suffix for consistency.

Reply:
Got it.